src/main/python/trec-covid/covid_baseline_tools.py

```diff
@@ -236,7 +236,7 @@ def evaluate_runs(qrels, runs, expected={}, check_md5=True):
                         'judged_cut_20', 'map', 'recall_1000', 'judged_cut_1000']:
                 if key in expected[run]:
                     assert metrics[key] == expected[run][key],\
```

Comment:
shouldn't this be noted with first four decimal places too?

Reply:
Addressed your comment with ` math.isclose`